cliquet/initialization.py

```diff
@@ -76,7 +76,8 @@ def setup_authentication(config):
     from configuration.
     """
     config.include('pyramid_multiauth')
-    config.set_default_permission('readwrite')
+    # By default, permissions are handled dynamically.
+    config.set_default_permission('dynamic')
```

Comment:
dynamic should be a constant like those http://docs.pylonsproject.org/projects/pyramid//en/latest/api/security.html#constants